kod/object/passive/spell/radiusench/bane.kod

```diff
@@ -0,0 +1,116 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+BaneEnchantment is RadiusEnchantment
+
+constants:
+
+   include blakston.khd
+
+resources:
+   
+   default_Hurt_Caster = "A tide of energy retaliates against your spell."
+   default_Kills_You = "The oncoming tide of energy carries you straight to "
```

Comment:
Don't capitalize resource names (i.e. make them look like the majority of what already exists). Also for a multi-line resource, follow this convention:

    resource_name = \
       "Line 1 "
       "line 2."